data/frameReceiver/src/PercivalEmulatorFrameDecoder.cpp

```diff
@@ -131,25 +129,11 @@ void PercivalEmulatorFrameDecoder::process_packet_header(size_t bytes_received,
 	uint8_t  subframe = get_subframe_number();
 	uint8_t  type = get_packet_type();
 
```

Comment:
Same comment as previous regarding the file name, is it time to update these files to be the actual files and remove the Emulator word from them?

Reply:
I think I have done it actually.